### ISSUE TITLE
README no longer display in repo browser

### ISSUE THREAD

**Issue by jrversteegh** [trac-hacks/trac-readme-plugin#4]
My README.md is displayed on the bottom of the repository "home" page (nice!!), but when I click the file, it's contents are no longer displayed. Trac 1.2.1, Genshi 0.7.

**Comment by rjollos** [comment]
Is this a new install, or did the behavior start when you upgraded Trac? Might have been fixed in 04ceb214.

**Comment by jrversteegh** [comment]
Trac has not been upgraded for some time. Was just trying out the plugin. It was egged from the latest source and installed through the Admin menu, so should include said revision.
I also tried changes to trac.ini, like stated in the readme:
```
mime_map = text/x-dylan:dylan,text/x-idl:ice,text/x-ada:ads:adb,**text/x-trac-wiki:wiki,text/x-markdown:md**
mime_map_patterns = text/plain:README(?!\.rst)|INSTALL(?!\.rst)|COPYING.*
```
which didn't change anything

**Comment by rjollos** [comment]
The reason I asked is that the issue sounds a bit like #2. I'm not able to reproduce. Are there any errors in the [browser console](https://trac.edgewall.org/wiki/TracTroubleshooting#ChecktheLogs)?

**Comment by rjollos** [comment]
Here are some screen captures that demonstrate the testing I've done. The first screen capture shows the `README.md` from the Repository Browser view, viewing the root of the repository contents. The second screen capture shows the FIle view of `README.md`.


![screen shot 2018-09-16 at 21 08 19](https://user-images.githubusercontent.com/1152089/45605810-c569bc80-b9f4-11e8-914a-7c97d3d88d7d.jpg)

----

![screen shot 2018-09-16 at 21 08 33](https://user-images.githubusercontent.com/1152089/45605811-c995da00-b9f4-11e8-8c03-5c933955d3d0.jpg)

**Comment by jrversteegh** [comment]
Browser console doesn't show anything. Neither does the apache error log nor trac.log.

Interesting to note that a "ReadMe.txt" in an svn repo is begin displayed, while README.md in my git repo is not.

**Comment by jrversteegh** [comment]
So looking at your fix for #2, it looks like it might be related to:
```
add_script(req, 'readme/marked.js')
add_script(req, 'readme/readme.js')
```

I'll try and have a closer look tomorrow.

**Comment by jrversteegh** [comment]
I don't want the readme's rendered when attempting to view their source anyway (except for maybe syntax highlighting), so https://github.com/orca-software/trac-readme-plugin/commit/38998fc6ef5636191a8f1f0871597cab13ebf603 fixed it for me.
